src/extension.ts

```diff
@@ -539,6 +570,31 @@ function launchMetals(
       }
     );
 
+    registerCommand("metals.analyze-stacktrace", () => {
```

Comment:
When this is merged, https://github.com/scalameta/metals-languageclient/pull/184, let's use it here.

Reply:
Fixed!